clusterloader2/pkg/measurement/common/ooms_tracker.go

```diff
@@ -163,25 +175,33 @@ func (m *clusterOOMsTrackerMeasurement) gather() ([]measurement.Summary, error)
 
 	m.stop()
 
-	content, err := util.PrettyPrintJSON(struct {
-		Ooms []oomEvent `json:"ooms"`
-	}{
-		Ooms: m.ooms,
-	})
+	oomData := make(map[string][]oomEvent)
+	oomData["failures"] = make([]oomEvent, 0)
+	oomData["past"] = make([]oomEvent, 0)
+	oomData["whitelisted"] = make([]oomEvent, 0)
 
+	for _, oom := range m.ooms {
+		if m.startTime.After(oom.Time) {
+			oomData["past"] = append(oomData["past"], oom)
+			continue
+		}
+		if whitelisted := m.processWhitelist[oom.Process]; whitelisted {
+			oomData["whitelisted"] = append(oomData["whitelisted"], oom)
+		} else {
```

Comment:
else is rarely used in go
if{
continue
}
normal case unindented

Reply:
Done.